core/templates/dev/head/domain/subtopic_viewer/subtopic-viewer-backend-api.service.ts

```diff
@@ -33,8 +40,11 @@ angular.module('oppia').factory('SubtopicViewerBackendApiService', [
 
       $http.get(subtopicDataUrl).then(function(response) {
         subtopicDataDict = angular.copy(response.data);
+        subtopicDataObject = subtopicDataObjectFactory.createFromBackendDict(
```

Comment:
Do we still need the `subtopicDataDict` variable?

Reply:
You are right, it is no longer needed, Removed it.